_boards_cm/techbase-modberry-500-cm4.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
@punnypenguins - I think this file might've snuck into this PR. Can you remove from this PR?